apps/demo/src/HsdsApp.tsx

```diff
@@ -19,13 +26,17 @@ function HsdsApp() {
   const [searchParams] = useSearchParams();
   const filepath = `${SUBDOMAIN}${searchParams.get('file') || FILEPATH}`;
 
+  const fetcher = useMemo(() => {
+    return createAxiosFetcher(
+      axios.create({
+        adapter: 'fetch',
+        auth: { username: USERNAME, password: PASSWORD },
```

Comment:
I had to keep the initialization of the `fetcher` inside the component. If I move it out, I have to move two of the `assertEnvVar` calls out as well, which then forces the `VITE_HSDS_USERNAME` and `VITE_HSDS_PASSWORD` env vars to be provided, which is annoying when running tests or the demo for the first time.